docs/guide/troubleshooting/faq.md

```diff
@@ -1,9 +1,10 @@
-# Coming Soon
+# FAQ
 
-::: info
-This page is Under Construction 🏗️, whilst Moonwall has been built - the docs have not.
+### How do I upgrade Moonwall?
 
-Please Stay tuned for more details of this feature!
-:::
+You can upgrade Moonwall with the following command:
```

Comment:
Can we add a note here to always check changelog for details of breaking changes?

Sometimes moonwall changes underlying client libraries (like ethers, or polkadotJs), which introduce changes to how we use their libaries. For example going from ethers v5 -> v6 meant going from using `BigNumber` to `bigint` types